src/preferences.cpp

```diff
@@ -0,0 +1,165 @@
+#include "preferences.h"
```

Comment:
Could you add license info for this file as well?

Reply:
I've added the license with: a68fe4a9f581b1bdf2b299cb383bb8ae871911ae